app/views/layouts/_doctype.html.haml

```diff
@@ -0,0 +1 @@
+!!! 5
```

Comment:
It's the "haml way" -- http://haml.info/docs/yardoc/file.REFERENCE.html#doctype_

Personally I do not care. Whatever gives us the best probability that we do not start using different doctypes (again) in different places is good for me